docs/companions.md

```diff
@@ -1,8 +1,9 @@
 # Companion Configuration
 
-Have a look at the [basic terminology](../README.md) to understand what a
-companion is. For these use cases the following sections provide example
-configurations.
+For better understanding, take a look at [Companions
+section](https://github.com/aixigo/PREvant?tab=readme-ov-file#companions) to
```

Comment:
The linking wouldn't work if you work on another branch or fork. Can you try to use relative links?

```suggestion
section](../README.md#companions) to
```